contracts/src/factory/HyperdriveCreate3Factory.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.22;
+
+import { CREATE3 } from "solmate/utils/CREATE3.sol";
+
+/// @author DELV
+/// @title HyperdriveCreate3Factory
+/// @notice Uses CREATE3 to deploy a contract to a precomputable address that
+///         is only dependent on the deployer's address and the provided salt.
```

Comment:
Let's add a `@dev` comment. This kind of deployment isn't suitable for high-volume contracts since it uses a proxy. I'm fine with using it for the registry, but every call will cost an additional 2700+ gas because of the extra call.